src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -1960,6 +1961,7 @@ class RsBlockExprStub(
             flags = BitUtil.set(flags, UNSAFE_MASK, psi.isUnsafe)
             flags = BitUtil.set(flags, ASYNC_MASK, psi.isAsync)
             flags = BitUtil.set(flags, TRY_MASK, psi.isTry)
+            flags = BitUtil.set(flags, TRY_MASK, psi.isConst)
```

Comment:
@mchernyavsky This looks like it should be `CONST_MASK` instead?

Reply:
Oh shit.
@kumbayo thanks!